composer.json

```diff
@@ -43,9 +43,8 @@
         "liip/functional-test-bundle": "~1.0",
         "sensio/generator-bundle": "~3.0",
         "incenteev/composer-parameter-handler": "~2.0",
-        "symfony/phpunit-bridge": "~3.0"
+        "symfony/phpunit-bridge": "^3.3"
```

Comment:
s/still/already? but yeah, whatever works for this, its only a dev requirement so if it builds fine on all supported platforms, lets use this version.